### DISCUSSION TITLE
💖 looking for new molecule azure driver maintainers

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible-community/molecule-plugins#35] [General]
This driver needs at least one maintainer as currently I cannot be really counted as one. I will help anyone interested with CI assistance and molecule core stuff.

**Comment by ghost** [comment]
I did not use this driver but thinking to use, so I can help

**Comment by ssbarnea** [comment]
Please be sure that you watch the project for all changes, perform reviews and ping me on irc if needed. After a few interactions, I will propose you to be added as core. I am more than happy to stop doing unreviewed merges due to the lack of peer-reviewers.

**Comment by ghost** [comment]
sure, will keep an eye on it
